examples/jsm/nodes/accessors/TextureNode.js

```diff
@@ -181,6 +189,15 @@ class TextureNode extends UniformNode {
 
 	}
 
+	compare( compareNode ) {
+
+		const textureNode = this.clone();
+		textureNode.compareNode = nodeObject( compareNode );
```

Comment:
Do we actually need `nodeObject` here?

Reply:
It's needed for primitive values like `textureNode.compare( .5 )`